src/pages/dashboard/dashboard.html

```diff
@@ -0,0 +1,35 @@
+<ion-header>
+  <ion-navbar>
+    <ion-buttons start>
+      <button ion-button (click)="logout()" *ngIf="authenticationProvider.logoutEnabled()">
+        <span>Logout</span>
+        <h6 class="version-number">Version {{pageState.appVersion$ | async}}</h6>
+      </button>
+      <h6 *ngIf="!authenticationProvider.logoutEnabled()" class="version-number">Version {{pageState.appVersion$
```

Comment:
Could use a function `isLogoutEnabled()` instead of using the provider in the html template.

Reply:
I think this was just a cut and paste of the code from the journal. A function would be neater.

Reply:
This is done.